.github/workflows/pr-title.yml

```diff
@@ -49,4 +49,4 @@ jobs:
           # will suggest using that commit message instead of the PR title for the
           # merge commit, and it's easy to commit this by mistake. Enable this option
           # to also validate the commit message for one commit PRs.
-          validateSingleCommit: true
+          validateSingleCommit: false
```

Comment:
Should this not be true?

Reply:
Thanks, that's a bug copy-pasted from the template. I'll fix it there also. 

It's a [legacy configuration](https://github.com/amannn/action-semantic-pull-request#legacy-configuration) but I'll set it back to true, and add the other one that's mentioned.